stages/org.osbuild.ostree.deploy.container

```diff
@@ -109,6 +120,25 @@ def ostree_container_deploy(tree, inputs, osname, target_imgref, kopts):
             ostree.cli("container", "image", "deploy",
                        *extra_args, sysroot=tree, *kargs)
 
+            aleph_image_digest = skopeo_cli("inspect", f"{image_source}", "--format", '"{{ .Digest }}"')
+            aleph_image_revision = skopeo_cli("inspect", f"{image_source}", "--format", '"{{ index .Labels \"org.opencontainers.image.revision\" }}"')
+            aleph_image_source = skopeo_cli("inspect", f"{image_source}", "--format", '"{{ index .Labels \"org.opencontainers.image.source\" }}"')
+            aleph_image_version = skopeo_cli("inspect", f"{image_source}", "--format", '"{{ index .Labels \"org.opencontainers.image.version\" }}"')
+
+            aleph_version_data = {
+                "osbuild-version": osbuild.__version__,
+                "image-name": image_name,
+                "image-digest": aleph_image_digest,
+                "image-revision": aleph_image_revision,
+                "image-source": aleph_image_source,
+                "image-version": aleph_image_version
+            }
+
+            aleph_json = json.dumps(aleph_version_data, indent=4)
+            with open(tree + "/.coreos-aleph-version.json", "w") as outfile:
```

Comment:
It's not always "coreos" here though in general...
Could call it `/.osbuild-container-aleph.json` e.g. much how bootc namespaces [its version](https://github.com/containers/bootc/blob/82a6a5a570277c54f027ae19f8da89030ab9de0b/lib/src/install.rs#L223)